test/integration/default/controls/_mapdata.rb

```diff
@@ -2,8 +2,8 @@
 
```

Comment:
Just remembered that this change has knock-on effects.

This line needs to be updated:

https://github.com/saltstack-formulas/template-formula/blob/978a7e7cd04c00fe6e7b5d113926683a86534094/CODEOWNERS#L21

And when this is merged, please remind me to update everything related in the `ssf-formula`!

* https://github.com/myii/ssf-formula/search?q=spec.rb

Reply:
Thanks, I didn't thought about it.

Reply:
It's now renamed in `CODEOWNERS`. Thanks.